karate-tests/src/test/java/patients/appRestricted/getPatient.feature

```diff
@@ -0,0 +1,97 @@
+Feature: get /Patient - Application-restricted access mode
```

Comment:
Is this set of tests meant to cover the "Get patient details" operation (https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir#get-/Patient/-id- ) or "Search for a patient"?

Reply:
It does both, but it will be reviewed in due course when the real work of adding tests is underway. For now, it's really just there to show we can authenticate and do useful stuff.